build.gradle

```diff
@@ -9,37 +13,44 @@ repositories {
 dependencies {
     compile gradleApi()
     compile localGroovy()
-    compile files('../../libs/umple-latest.jar')
+    compile files('../../libs/umple.jar')
 
     testCompile group: 'junit', name: 'junit', version: '4.11'
 }
 
-apply plugin: 'maven'
-
-group 'cruise.umple'
-version '0.0.0'
-
-uploadArchives {
-    repositories {
-        mavenDeployer {
-            repository(url: uri('../libs/'))
-        }
-    }
-}
-
 // First, apply the publishing plugin
 buildscript {
   repositories {
     maven {
       url "https://plugins.gradle.org/m2/"
     }
+	jcenter()
   }
   dependencies {
     classpath "com.gradle.publish:plugin-publish-plugin:0.9.6"
+	classpath 'de.undercouch:gradle-download-task:3.1.2'
   }
 }
 
+apply plugin: 'de.undercouch.download'
 apply plugin: "com.gradle.plugin-publish"
+apply plugin: 'maven'
+
+download {
+	onlyIfNewer = true
+	overwrite = true
+    src 'http://cruise.eecs.uottawa.ca/umpleonline/scripts/umple.jar'
+    dest '../../libs/'
```

Comment:
I really don't like this, you're assuming that there is a "nested" file structure.